src/elfcore.c

```diff
@@ -157,13 +168,15 @@ typedef struct elf_timeval {    /* Time value with microsecond resolution    */
   long tv_usec;                 /* Microseconds                              */
 } elf_timeval;
 
-
+#if !defined(__aarch64__)
 typedef struct elf_siginfo {    /* Information about signal (unused)         */
   int32_t si_signo;             /* Signal number                             */
   int32_t si_code;              /* Extra code                                */
   int32_t si_errno;             /* Errno                                     */
 } elf_siginfo;
-
+#else
+typedef struct elf_siginfo elf_siginfo;
+#endif
```

Comment:
Strange.  Why isn't this defined on Intel?

Reply:
> Strange. Why isn't this defined on Intel?

It is defined for Intel too in `<sys/procfs.h>` which was not included directly. But it was picked up on arm64 somehow.
I will include `<sys/procfs.h>` and remove this `#if`